### PR TITLE
SinaiEt 006 Abb 105

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Sinai Et 006 new, Abb 105 minor changes. For the moment I used the old record for Weddase Maryam.

### CODE REVIEW

**Review by eu-genia — APPROVED** [review]

File: ParisBNF/abb/BNFabb105.xml [hunk 1]
```diff
@@ -935,20 +935,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-                        mentioned in supplications on some folia (e.g. <ref target="#139ra #143ra"></ref>) might have been the scribe.
+                        mentioned in supplications on some folia (e.g. <ref target="#139ra #143ra"></ref>) might have
+                        been the scribe who copied the text.
                      </origin>
                      <provenance>The manuscript was bought by
                        <persName ref="PRS1071dAbbadi" role="owner">Antoine d'Abbadie</persName>
                        sometime between <date>1840</date> and <date>1848</date>.
                      </provenance>
-                     <acquisition>The manuscript entered the Bibliothèque nationale in
+                     <acquisition>The manuscript entered<placeName ref="INS0303BNF"/> in
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb105.xml:949]: you need a space before the tag, and no space inside of the tag before the /

File: SinaiStCatherine/SinaiEt006.xml [hunk 2]
```diff
@@ -0,0 +1,707 @@
...
+      ዘኪሩቤል፡ ሰማይ፡ ይጸርኁ፡ ወይሰጠዉ፡ ቃለ፡ በኩራ፡ ለድንግል፡ ለጻድቃን፡ ክብር።
+      <gap reason="ellipsis" resp="DN"/><pb n="3r"/>
+      <gap reason="lost" extent="20" unit="charcters"/> ሕፃን፡ ንኡስ፡ ቂርቆስ፡
+      ቅዱስ፡ ነቅዕ፡ ዘእምዐዘቅት፡ ማይ፡ ዘኢይማስን፡
+    </incipit>
+ </msItem>
+
+ <msItem xml:id="ms_i8">
+ <locus from="211v" to="212r"/>
+ <title type="incomplete">A portion of the pascal tables (one table only),
```
Comment (eu-genia) [inline SinaiStCatherine/SinaiEt006.xml:275]: Computational tables for calculating the day of Easter? (never heard of pascal tables, unless they are coded in the Pascal language...)

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: SinaiStCatherine/SinaiEt006.xml [hunk 3]
```diff
@@ -0,0 +1,707 @@
...
+
+                       </item>
+
+                       <item xml:id="a4">
+                         <locus target="#2v"/>
+                         <desc type="CommemorativeNote">A note recording the
+                           death of <persName ref="PRS13951Petros">the priest Ṗeṭros</persName>
+                            that took place on <date>14 Gǝnbot 6952
+                           "year of mercy" (=1444 CE)</date>.</desc>
+                         <q xml:lang="gez">በ፷፻፡ ወበ፱፻፶ወ፪፡ አመተ፡ ምሕረት፡ አዕረፈ፡ ቀሲስ፡
```
Comment (thea-m) [inline SinaiStCatherine/SinaiEt006.xml:564]: you could fully mark up the date

File: SinaiStCatherine/SinaiEt006.xml [hunk 4]
```diff
@@ -0,0 +1,707 @@
...
+                         ጴጥሮስ፡ አመ፡ ፲ወ፬ ለግንቦት፡ በግእዝ፡ ወበሮሜ፡ ፱። ይምሐሮ፡ እግዚአብሔር፡ ወይደምሮ፡ ምስለ፡
+                       ቅዱሳኒሁ፡ አሜን፡ በጸሎታ፡ ለእግዚእትነ፡ ማርያም።</q>
+                         <note></note>
+                       </item>
+
+                       <item xml:id="a5">
+                         <locus target="#38r"/>
+                         <desc type="Unclear">Commentary or correction to the text of the Psalm.</desc>
+                         <q xml:lang="gez"></q>
+                         <note></note>
```
Comment (thea-m) [inline SinaiStCatherine/SinaiEt006.xml:574]: delete note

File: SinaiStCatherine/SinaiEt006.xml [hunk 5]
```diff
@@ -0,0 +1,707 @@
...
+         <langUsage>
+           <language ident="en">English</language>
+           <language ident="gez">Gǝʿǝz</language>
+           <language ident="grc">Greek</language>
+           <language ident="ar">Arabic</language>
+           <language ident="am">Amharic</language>
+         </langUsage>
+      </profileDesc>
+      <revisionDesc>
+         <change who="DR" when="2022-10-18">Created record from catalogue description with consultation of digitized microfilm</change>
```
Comment (thea-m) [inline SinaiStCatherine/SinaiEt006.xml:699]: I do not remember having done that ;)

**Review by thea-m — CHANGES_REQUESTED** [review]
Nice one :)

File: ParisBNF/abb/BNFabb105.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb105.xml:949]: ok

File: SinaiStCatherine/SinaiEt006.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply SinaiStCatherine/SinaiEt006.xml:275]: ok

File: SinaiStCatherine/SinaiEt006.xml [hunk 6]
```diff
@@ -0,0 +1,707 @@
...
+                    <locus from="3r" to="122vb"/>
+                     <title type="complete" ref="LIT5852Psalter"/>
+                     <textLang mainLang="gez"/>
+
+
+
+                    <msItem xml:id="ms_i1.1">
+                      <locus from="5r" to="8r"/>
+                      <title type="incomplete" ref="LIT6719prologue"/>
+                      <textLang mainLang="gez"/>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply SinaiStCatherine/SinaiEt006.xml:44]: It is complete, sorry.

File: SinaiStCatherine/SinaiEt006.xml [hunk 7]
```diff
@@ -0,0 +1,707 @@
...
+                   ፈሪሀ፡ እግዚአብሔር። ወምክር፡ ሠናይት፡ ለኩሉ፡ ዘይገብራ፡ ወስብሐቲሁኒ፡ ይነብር፡ ለዓለመ፡ ዓለም፡ አሜን።
+                    ሐሳብ፡ ቀዳሚ፡ ዘነገሩ፡ አበዊነ፡ ዘከመ፡ ለሐኮሙ፡ ለአቡነ፡ አዳም፡ ወለሔዋን፡</incipit>
+                     <note>Probably incomplete, the text on <locus target="#199vb"/>
+                   is erased or cannot be read in the microfilm.</note>
+           </msItem>
+
+                   <msItem xml:id="ms_i4">
+                       <locus target="#200r"/>
+                       <title type="complete" ref="LIT6011FeastsList">A version of the list of church feasts,
+                       with the names of the months in Arabic transcribed in Ethiopic, starting with
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply SinaiStCatherine/SinaiEt006.xml:226]: Ok, thanks.

File: SinaiStCatherine/SinaiEt006.xml [hunk 8]
```diff
@@ -0,0 +1,707 @@
...
+                     <foreign xml:lang="gez">ኤሎል</foreign></title>
+                       <textLang mainLang="gez"/>
+                    </msItem>
+
+                <msItem xml:id="ms_i5">
+                  <locus from="200r" to="203v"/>
+               <title type="complete" ref="LIT6291HassabaRetu">A version of the treatise on computus and calendar that sometimes bear
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply SinaiStCatherine/SinaiEt006.xml:233]: I shortened it a bit.

File: SinaiStCatherine/SinaiEt006.xml [hunk 9]
```diff
@@ -0,0 +1,707 @@
...
+    <incipit xml:lang="gez"><gap reason="lost" extent="6" unit="characters"/>
+    ስምከ፡ ሕያው፡ ዘኢይመውት፡ ትጉህ፡ <gap reason="lost" extent="3" unit="characters"/> ዘኢይደክም፡
+      እግዚእ፡ ስሉጥ፡ ለዕለ፡ ኵሉ፡ አምላክ፡ ዘኢይትኀባእ፡ ዘኪያከ፡ ያንቀያዓዱ፡ ኵሉ፡ ፍጥረት። ዘበሰማይ፡ ወዘበ፡ ምድር፡ ዘፈጠርከ፡
+  </incipit>
+   </msItem>
+
+   <msItem xml:id="ms_i7">
+  <locus from="208r" to="211v"/>
+  <locus from="3r" to="4v"/>
+  <title type="incomplete" ref="LIT1990Mawase">Excerpts from the Mawāśǝʾǝt
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply SinaiStCatherine/SinaiEt006.xml:259]: ok

File: SinaiStCatherine/SinaiEt006.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply SinaiStCatherine/SinaiEt006.xml:275]: Reshaped.

File: SinaiStCatherine/SinaiEt006.xml [hunk 10]
```diff
@@ -0,0 +1,707 @@
...
+                           <dimensions type="outer" unit="mm">
+                              <height>195</height>
+                              <width>140</width>
+                            <!-- <depth></depth>-->
+                           </dimensions>
+                        </extent>
+                        <foliation>Foliation in the middle of
+                          the upper margin.</foliation>
+                        <collation>
+                          <signatures></signatures>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply SinaiStCatherine/SinaiEt006.xml:302]: ok

File: SinaiStCatherine/SinaiEt006.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply SinaiStCatherine/SinaiEt006.xml:564]: ok

File: SinaiStCatherine/SinaiEt006.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply SinaiStCatherine/SinaiEt006.xml:574]: ok

File: SinaiStCatherine/SinaiEt006.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply SinaiStCatherine/SinaiEt006.xml:699]: :)

**Pull request merged**